dags/collection_generator.py

```diff
@@ -73,7 +74,8 @@
                             },
                             # {"name": "TRANSFORMED_JOBS", "value": str('{{ task_instance.xcom_pull(task_ids="configure-dag", key="transformed-jobs") | string }}')},
                             {"name": "TRANSFORMED_JOBS", "value":"'{{ task_instance.xcom_pull(task_ids=\"configure-dag\", key=\"transformed-jobs\") | string }}'"},
-                            {"name": "DATASET_JOBS", "value": "'{{ task_instance.xcom_pull(task_ids=\"configure-dag\", key=\"dataset-jobs\") | string }}'"}
+                            {"name": "DATASET_JOBS", "value": "'{{ task_instance.xcom_pull(task_ids=\"configure-dag\", key=\"dataset-jobs\") | string }}'"},
+                            {"name": "INCREMENTAL_LOADING_OVERRIDE", "value": "'{{ \"yes\" if task_instance.xcom_pull(task_ids=\"configure-dag\", key=\"incremental-loading-override\") else \"\" }}'"}
```

Comment:
Do you need a conditional around this?  Feels like since you've defined a default anyway, you can just take the literal value.  Hopefully it wouldn't even need conversion to a yes/no string either?